test/integration/recover_remote_nse_nsm_heal_test.go

```diff
@@ -19,6 +19,7 @@ import (
 
 func TestNSMHealRemoteDieNSMD_NSE(t *testing.T) {
 	RegisterTestingT(t)
+	t.Skip("Skipped for a while, will be enabled soon")
```

Comment:
You can use `IsBrokeTestsEnabled` to check for the env variable. Then someone can run locally `make k8s-integration-TestNSMHealRemoteDieNSMD_NSE-test` and debug/test.